control-plane/tests/test_service_persistence_action_reconciliation_create_tracking_ticket.py

```
@@ -0,0 +1,1037 @@
+                ),
+                stale_after=support.datetime(
+                    2026,
+                    4,
+                    18,
+                    5,
+                    0,
+                    tzinfo=support.timezone.utc,
+                ),
+            )
+    def test_service_records_execution_correlation_mismatch_states_separately(self) -> None:
+        store, _ = make_store()
```

Ruff (0.15.10)

[error] Undefined name `make_store`

(F821)